app/src/main/java/com/powerhungers/fomezero/base_app/feature/login/domain/repository/LoginRepository.kt

```diff
@@ -0,0 +1,10 @@
+package com.powerhungers.fomezero.base_app.feature.login.domain.repository
+
+import com.powerhungers.fomezero.base_app.feature.login.domain.model.User
+import kotlinx.coroutines.flow.Flow
+
+interface LoginRepository {
+
+    fun login(email: String, password: String): Flow<User>
+
+}
```

Comment:
```suggestion
interface LoginRepository {
    fun login(email: String, password: String): Flow<User>
}
```
espaços desnecessarios